src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figures {
+    private int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double area() {
+        return 4 * side;
+    }
+
+    @Override
+    public void draw() {

```

Comment:
This method should be part of an interface, not implemented directly in this class. The `draw()` method is not conceptually close to the `area()` method, so they should not be in the same interface.